embedly/api.py

```diff
@@ -0,0 +1,14 @@
+"""Embedly proxy API"""
+import requests
+
+from django.conf import settings
+
+EMBEDLY_URL = "https://api.embed.ly/1/oembed"
```

Comment:
I would suggest putting this into `settings.py` with this as the default so in case it changes on us it doesn't require a code change.